lib/manageiq/providers/openstack/legacy/events/openstack_rabbit_event_monitor.rb

```diff
@@ -0,0 +1,156 @@
+require 'manageiq/providers/openstack/legacy/openstack_event_monitor'
+require 'manageiq/providers/openstack/legacy/events/openstack_event'
+require 'bunny'
```

Comment:
You need to add 'bunny' to your gemspec, and probably remove it from `manageiq-gems-pending`.